services/web/client/source/class/osparc/desktop/SlideShowView.js

```diff
@@ -150,20 +148,37 @@ qx.Class.define("osparc.desktop.SlideShowView", {
       });
     },
 
-    __next: function() {
+    __syncButtons: function() {
       const study = this.getStudy();
       if (study) {
-        const slideShow = study.getUi().getSlideshow();
-        const nodes = [];
-        for (let nodeId in slideShow) {
-          const node = slideShow[nodeId];
-          nodes.push({
-            ...node,
-            nodeId
-          });
+        const nodes = this.self().getSortedNodes(study);
+        if (nodes.length && nodes[0].nodeId === this.__currentNodeId) {
+          this.__prvsBtn.setEnabled(false);
```

Comment:
nice - one small thing, would it be possible to make it so that whenever you click on up/down for node order that you automatically highlight that node as the "selected one" so that you see it move? This already happens if you click on the node name and then the up/down buttons but if I have one selected and click another node's up/down, the highlight stays with the original node's. I leave this up to your judgement.

Reply:
If there is no item selected, any action will automatically activate that entry. If there was already a selection, it will remain selected even though other items get actioned...